### PR TITLE
JavaScript example

### PR DESCRIPTION
**Pull request by Victori0us**

### CODE REVIEW

File: javascript/js/boot.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  init: function(options) {
+    HermesApp.Config = {
+      applicationTitle: "Hermes",
+      applicationUrl:   "http://localhost:5555",
+      perPage:          20,
+      dateTimeFormat:   "YYYY-MM-DD HH:mm:ss"
+    };
+
+    HermesApp.Data = {
+      saltedgeBaseUrl:   "https://banksalt.com",
```
Comment (idanci) [inline javascript/js/boot.js]: saltedge.com

**Comment by idanci** [comment]
Please add README file

File: javascript/js/boot.js (hunk 1 above)
Comment (Victori0us) [inline javascript/js/boot.js]: for check need staging

**Comment by Victori0us** [comment]
ReadMe will be after merge to work with pictures

File: javascript/js/models/customer.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    params = {
+      data: {
+        customer_id: this.get("customer_id")
+      }
+    };
+
+    return $.ajax({
+      beforeSend: (function(_this) {
+        return function(xhrObj) {
+          xhrObj.setRequestHeader("Accept", "application/json");
```
Comment (alisnic) [inline javascript/js/models/customer.js]: extract header logic to a separate method

File: javascript/js/models/login.js [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+        model = collection.get(params.id);
+
+        if (model) {
+          modelAttributes = model.attributes || {};
+          _.defaults(modelAttributes, params);
+        }
+
+        _this.customer.set(collection.length ? {
+          logins: collection.models
+        } : void 0, {
```
Comment (alisnic) [inline javascript/js/models/login.js]: extract condition to a variable

File: javascript/js/views/connect/connect.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      return HermesApp.redirect("/dashboard");
+    }
+  },
+
+  onDestroy: function() {
+    return Backbone.$(window).off("message");
+  },
+
+  onMessageEvent: function(event, data) {
+      var message, view;
```
Comment (alisnic) [inline javascript/js/views/connect/connect.js]: fix intendation

File: javascript/vendor/css/bootstrap/css/bootstrap-theme.css [hunk 5]
```diff
@@ -0,0 +1,587 @@
+/*!
```
Comment (alisnic) [inline javascript/vendor/css/bootstrap/css/bootstrap-theme.css]: fetch this css from a CDN, do not include it in the pull-request

File: javascript/vendor/js/backbone.babysitter.js [hunk 6]
```diff
@@ -0,0 +1,190 @@
+// Backbone.BabySitter
```
Comment (alisnic) [inline javascript/vendor/js/backbone.babysitter.js]: load from CDN and remove from pull request

**Pull request merged**